ngraph/core/src/op/max.cpp

```diff
@@ -96,7 +96,6 @@ namespace maxop
                   const AxisSet& axes,
                   bool keep_dims)
     {
-        out->set_shape(reduce(arg->get_shape(), axes, false));
```

Comment:
How does this removal affect the behavior? Are you sure this line is not needed?

Reply:
There was an issue with this shape setting, and most likely it was removed to make tests pass. But I nailed this issue which was caused by hardcoded keep_dims = false for reduce call. 
So I returned it. But anyway, I suppose that this set_shape not really required, because interpreter has to handle shapes (Maybe this call actual only for dynamic shapes?).